wee_slack.py

```diff
@@ -2576,8 +2608,16 @@ def subprocess_channel_leave(message_json, eventrouter, channel, team):
 
 
 def subprocess_message_replied(message_json, eventrouter, channel, team):
-    pass
-
+    parent_ts = message_json["message"].get("thread_ts")
+    parent_message = channel.messages.get(SlackTS(parent_ts))
+    # Thread exists but is not open yet
+    if parent_message is not None \
+            and not (parent_message.thread_channel and parent_message.thread_channel.active):
+        channel.hash_message(parent_ts)
+        if message_json["message"].get("user") == team.myidentifier:
+            parent_message.notify_thread(action="response")
+        elif any(team.myidentifier == r["user"] for r in message_json["message"]["replies"]):
+            parent_message.notify_thread(action="participant")
```

Comment:
I changed, but I’m not happy with the solution which feels hacky. If you have a better one...

Reply:
Ugh, I took it for granted that the `message_replied` event had a field for the user that wrote the message, but I guess not...

I don't see any other way this can be checked from the `message_replied` event. I would think the replies array is sorted, but maybe it's best like you did to be safe. You have to check it for both cases though, so place it with the first if (before hash_message).

Reply:
You will have to do a similar check in `subprocess_thread_message`. There you can just check the user of the message though.

Reply:
Done in both cases. I’m wondering wether the check should be moved to the notify_thread method, but that would mean passing the thread message as a parameter to the method, which seems wrong

Reply:
What about passing just the user to notify_thread?

Reply:
ok